test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1415,7 +1415,8 @@ def forward(self, x, update):
         self.run_test(CopyModel(), (x, update))
 
     @skipIfUnsupportedMinOpsetVersion(11)
-    @disableScriptTest()  # Missing input size (with ellipsis indexing)
+    # TODO: Limited scripting support with ellipsis indexing.
+    #       Due to dependency on input tensor rank being known.
```

Comment:
does this mean test will pass only when shape_inference is turned on?

Reply:
no, shape inference won't affect this case. I will add stricter test case which require shape inference, with the PR of enabling passes dependent on shape/type info.